src/darwin/Framework/CHIP/templates/availability.yaml

```
@@ -7803,8 +7803,12 @@
               Feature:
                   DeadFrontBehavior: DeadFront
 
-- release: "Future"
-  versions: "future"
+- release: "Winter 2024"
```

```suggestion
- release: "Spring 2024"
```
Should this be Spring?

---

Per Slack discussion: "Early 2024"